### ISSUE TITLE
Investigation into the Iraq war: showing as 'not enough information'

### ISSUE THREAD

**Issue by MyfanwyNixon** [mysociety/theyworkforyou#1020]
Tweet from a user: https://twitter.com/ochayethenews/status/672283810101374980

Looks as if we are not calculating a position on this stance, although we do in fact have enough information to do so.

**Comment by struan** [comment]
Mmmm, the code at the TheyWorkForYou end is doing the correct thing in that none of the votes are listed as being strong ( which is what it needs to make a decision about which way someone votes ).

We'll need to update the votes on PublicWhip to make some of them strong.

**Comment by RichardTaylor** [comment]
I have reviewed the votes which make up this policy and have made a number of them into "strong" votes which will mean a stance will be shown on relevant MPs pages on TheyWorkForYou. (Once TheyWorkForYou picks up the change). 

Single sentence summaries have been added to the votes which contribute to the policy so pages like:

http://www.theyworkforyou.com/mp/11309/tom_watson/west_bromwich_east/divisions?policy=975

will become more informative. 

Many of the votes are still marked minor for example where question was one of timing of a [further] inquiry rather than the principle of an inquiry.
